meta-lmp-base/classes/lmp-staging.bbclass

```diff
@@ -5,15 +5,15 @@
 #
 # Copyright 2022 (C) Foundries.IO LTD
 
-INHERIT_KERNEL_MODSIGN = ""
+LMPSTAGING_INHERIT_KERNEL_MODSIGN = ""
```

Comment:
Could you expand the commit message with an explanation of why we need these changes?

Reply:
The idea is to clarify that these variables are used in this classe. I will add it to the commit